### ISSUE TITLE
List/grid information

### ISSUE THREAD

**Issue by RuturajL** [nvaccess/nvda#18145]
### Steps to reproduce:
Observe NVDA announcements in Start menu or even on desktop. 
### Actual behavior:
It announces "Desktop List", or, upon pressing windows key and pressing down arrow, it announces "Pinned list".
### Expected behavior:
Given that it's actually a grid of icons, which all experienced users will be aware of, NVDA should announce it as such.
Announcing it as list may lead to users using just down/up arrow, potentially missing out on some icons. Although in that case, NVDA's announcement scheme, Icon name + 1 of 20 then, Icon name + 4 of 20, that jump in numbers should provide some hint.

#### NVDA installed/portable/running from source:
installed.
#### NVDA version:

Version: 2024.4.2 (2024.4.2.35031)
#### Windows version:
Windows 11 24H2 (AMD64) build 26100.4061
#### Impact
Impact is Neglagable. It may be more important on touchscreen devices. Google talkback and Voiceover do announce this information correctly. I was unable to test with Narrator or Jaws currently.
